.storybook/stories/ThanksPageUnifiedVersion.stories.js

```diff
@@ -0,0 +1,102 @@
+import ThanksPageUnifiedVersion from '#src/components/Thanks/ThanksPageSingleVersion';
+import apolloStoryMixin from '../mixins/apollo-story-mixin';
+import cookieStoreStoryMixin from '../mixins/cookie-store-story-mixin';
+import {
+  mockedReceiptData,
+  mockLender,
+  mockLoans,
+  mockUserAchievementProgress,
+  mockContentful,
+  MOCK_TIERED_BADGE_ID,
+} from '../mock-data/thanks-badges-mock-data';
+
+export default {
+  title: 'Page/ThanksPageUnifiedVersion',
+  component: ThanksPageUnifiedVersion,
+};
+
+const queryResult = {
+  data: {
+    userAchievementProgress: mockUserAchievementProgress,
+    contentful: mockContentful,
+  }
+};
+
+const mockTieredBadge = { achievementId: MOCK_TIERED_BADGE_ID };
+
+const story = (args = {}, result = queryResult) => {
```

Comment:
It looks like Roger somehow only beat you by a min to open his PR. Will this story work well to show the module he created as well?